modules/apiserversource-kn.adoc

```diff
@@ -0,0 +1,162 @@
++
+[NOTE]
+====
+If you want to re-use an existing service account with the appropriate permissions, you must modify the `authentication.yaml` for that service account.
+====
++
+Create the service account, cluster role and cluster binding by entering the following command:
++
+----
+$ oc apply --filename authentication.yaml
+----
+
+. Create an ApiServerSource event source by entering the following command:
++
+----
+$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa
+----
+
+. To check that the ApiServerSource is set up correctly, create a Knative service that dumps incoming messages to its log by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display
+----
+
+. Create a trigger from the `default` broker to filter events to the service created in the previous step by entering the following `kn` command:
++
+----
+$ kn trigger create event-display-trigger --broker default --sink svc:event-display \ source=dev.knative.samples/testevents \
+----
+
+. Create events by launching a pod in the default namespace.
+
+.Verification steps
+
+You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+
+You can view the message dumper function logs by entering the following commands:
+----
+$ oc get pods
+$ oc logs -l serving.knative.dev/service=event-display -c user-container
+----
+
+The logs should contain lines similar to the following:
+----
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.apiserver.resource.update
+  source: https://10.96.0.1:443
+  subject: /apis/v1/namespaces/default/events/testevents.15dd3050eb1e6f50
+  id: e0447eb7-36b5-443b-9d37-faf4fe5c62f0
+  time: 2019-12-04T14:09:30.917608978Z
+  datacontenttype: application/json
+Data,
+  {
+    "apiVersion": "v1",
+    "count": 1,
+    "eventTime": null,
+    "firstTimestamp": "2019-05-10T23:27:06Z",
+    "involvedObject": {
+      "apiVersion": "v1",
+      "fieldPath": "spec.containers{busybox}",
+      "kind": "Pod",
+      "name": "busybox",
+      "namespace": "default",
+      "resourceVersion": "28987493",
+      "uid": "1efb342a-737b-11e9-a6c5-42010a8a00ed"
+    },
+    "kind": "Event",
+    "lastTimestamp": "2019-05-10T23:27:06Z",
+    "message": "Started container",
+    "metadata": {
+      "creationTimestamp": "2019-05-10T23:27:06Z",
+      "name": "busybox.159d7608e3a3572c",
+      "namespace": "default",
+      "resourceVersion": "506088",
+      "selfLink": "/api/v1/namespaces/default/events/busybox.159d7608e3a3572c",
+      "uid": "2005af47-737b-11e9-a6c5-42010a8a00ed"
+    },
+    "reason": "Started",
+    "reportingComponent": "",
+    "reportingInstance": "",
+    "source": {
+      "component": "kubelet",
+      "host": "gke-knative-auto-cluster-default-pool-23c23c4f-xdj0"
```

Comment:
@matzew do we really want a Google GKE reference here ? ;-)

@abrennan89 I would abbreviate the output here to only provide the headings and then "..." as content will definitely look different for the user (many data found here is specific to the cluster where the event occured), someting like:

```
☁️  cloudevents.Event
Validation: valid
Context Attributes,
  specversion: 1.0
  type: dev.knative.apiserver.resource.update
  datacontenttype: application/json
  ...
Data,
  {
    "apiVersion": "v1",
    "involvedObject": {
      "apiVersion": "v1",
      "fieldPath": "spec.containers{busybox}",
      "kind": "Pod",
      "name": "busybox",
      "namespace": "default",
       .....
    },
    "kind": "Event",
    "message": "Started container",
    "metadata": {
      "name": "busybox.159d7608e3a3572c",
      "namespace": "default",
      ....
    },
    "reason": "Started",
    ...
  }
```